Cargo.toml

```diff
@@ -90,3 +91,6 @@ fuels-core = { version = "0.47.0", path = "./packages/fuels-core", default-featu
 fuels-macros = { version = "0.47.0", path = "./packages/fuels-macros", default-features = false }
 fuels-programs = { version = "0.47.0", path = "./packages/fuels-programs", default-features = false }
 fuels-test-helpers = { version = "0.47.0", path = "./packages/fuels-test-helpers", default-features = false }
+
+[workspace.metadata.versions-replacer.external-versions]
+fuels-types = "0.35.3"
```

Comment:
`fuels-types` here isn't needed. It is used erroneously in one place pointing to https://docs.rs/fuel-types/0.35.3/fuel_types/struct.Address.html. 

The version can be taken from `fuel-core` which is one of the workspace deps.

Reply:
According to `Cargo.toml`, `fuel-core = "0.20.4"` while things like `fuel-{tx,vm} = "0.35.3"`. Am I missing something?

Reply:
Sorry, misread you :) Links in [53ebf5c](https://github.com/FuelLabs/fuels-rs/pull/1098/commits/53ebf5c864d8c2419a0bba2cd8972cc5cb3759c0) are now using `fuel-types` (not `fuel*s*-types`) and `external-versions` are unnecessary.